### PR TITLE
Included ebeetask source and firefly-e organization for ebeetask connector

### PR DESCRIPTION
**Pull request by dconvers1979**
Included ebeetask source and firefly-e organization for ebeetask connector

### CODE REVIEW

**Comment by dconvers1979** [comment]
I signed it!

File: organizations/firefly_e.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+ "id": "FIREFLYE",
```
Comment (pfrisella) [inline organizations/firefly_e.json]: this should match filename. So either change this to FIREFLY_E or rename the file to fireflye.json

File: sources/ebeetask.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "EBEETASK",
+  "name": "ebeeTask",
+  "categories": ["DATABASE"],
```
Comment (pfrisella) [inline sources/ebeetask.json]: If there is a more appropriate category to help users you can add it to https://github.com/googledatastudio/ds-data-registry/blob/master/categories.json

**Comment by dconvers1979** [comment]
Hi, I commited the changes but don't know if they are included on the pull request...

**Comment by pfrisella** [comment]
The id was updated.
You didn't say change anything related to source category comment I left. However, it's up to you, I just wanted to check that DATABASE is the best category and whether you think another category might be more helpful for users to find this source. If you're ok with only having DATABASE then this looks good and I can merge. Let me know.

**Comment by dconvers1979** [comment]
Yeah, database is fine... our system gathers information and stores it. The information gathers is used to build the reports.

Thanks for every thing

Diego Convers
Enviado desde iphone !!!!


El 19/06/2018, a la(s) 10:05 a. m., Pete Frisella <notifications@github.com<mailto:notifications@github.com>> escribió:


The id was updated.
You didn't say change anything related to source category comment I left. However, it's up to you, I just wanted to check that DATABASE is the best category and whether you think another category might be more helpful for users to find this source. If you're ok with only having DATABASE then this looks good and I can merge. Let me know.

**Comment by pfrisella** [comment]
LGTM. Thanks for the contribution!

**Pull request merged**